.gitignore

```diff
@@ -9,7 +9,6 @@ tmp/
 
 # generated files
 deploy/**/metadata.yaml
-*_assets.go
```

Comment:
do not remove this - it already causes unexpected changes included in the commit in member operator.

Reply:
Going to revert this,
 but i didn't understand it completely, did you mean, it caused some changes since i removed it from member operator so i don't remove it from here..?

Reply:
the `template_assets.go` files are already present in our projects locally. When you removed the .gitignore line, then git automatically picks these files and adds them to the commit. We have to manually delete the `template_assets.go` files to make sure that they are not included in the commit. This is not obvious and a bit tedious work.